app/js/auth/auth_headers.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+function AuthHeaders($http) {
```

Comment:
Since this is already inside the `Auth` module, don't you think it would look better if it was called just `Headers`?